# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/10_mm-2.7-glibc.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/metamail/metamail.c.ewt	Mon Aug 26 17:16:59 1996
+++ mm2.7/src/metamail/metamail.c	Mon Aug 26 17:17:12 1996
@@ -48,6 +48,7 @@
 #ifndef AMIGA
 #ifdef SYSV
 #include <termio.h>
+#include <sys/ioctl.h>
 #include <unistd.h>
 #else /* SYSV */
 #include <sgtty.h>
